scripts/package.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+fail() {
+    printf "$1\n" >&2
+    exit 1
+}
+
+package() {
+    # Create directory for packaged Helm charts
+    dest="package/$1/v$(./get-version.sh)"
```

Comment:
`**/*.tgz` is already added in gitignore file

Reply:
It might be used by other scripts that prepare the `index.yaml`. Although the index file is moved to the target place, it would be good to explicitly ignore the `package` directory.